geonode/layers/api/serializers.py

```diff
@@ -71,6 +71,59 @@ class Meta:
     attribute = serializers.CharField(read_only=True)
 
 
+class FeatureInfoTemplateField(DynamicComputedField):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def get_attribute(self, instance):
+        if instance.use_featureinfo_custom_template and instance.featureinfo_custom_template:
+            return instance.featureinfo_custom_template
+        else:
+            _attributes = instance.attributes.filter(visible=True).order_by('display_order')
+            if _attributes.count():
+                _template = '<div>'
+                for _field in _attributes:
+                    _label = _field.attribute_label or _field.attribute
+                    _template += '<div class="row">'
+                    if _field.featureinfo_type == Attribute.TYPE_HREF:
+                        _template += '<div class="col-xs-6" style="font-weight: bold; word-wrap: break-word;">%s:</div> \
```

Comment:
I would consider putting in common the `<div>` tag since only the values are changing and use the `format` string to replace the values. Maybe something like:
```python
_div_structure = "<div class="{}" style="font-weight: bold; word-wrap: break-word;">%s:</div>"
...... other code ....
_template += _div_structure.format("col-xs-6") \
```
Handling HTML templates in the code is always a mess, let's try to make in common as many things as possible!

Reply:
I don't think the `div` is always the same one... however I will double check.

Reply:
the divs are all different, for the time being I'll leave like this.